src/VisualStudio/CSharp/Test/PersistentStorage/Mocks/ServiceBrokerMock.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Copy of https://devdiv.visualstudio.com/DevDiv/_git/VS.CloudCache?path=%2Ftest%2FMicrosoft.VisualStudio.Cache.Tests%2FMocks&_a=contents&version=GBmain
+// Do not edit unless necessary.
+
+using System;
+using System.Collections.Generic;
+using System.IO.Pipelines;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.ServiceHub.Framework;
+
+namespace Microsoft.CodeAnalysis.UnitTests.WorkspaceServices.Mocks
+{
+    internal class ServiceBrokerMock : IServiceBroker
```

Comment:
📝 These are stubs, not mocks. This should have been named `StubServiceBroker` and been placed in a `Fakes` namespace.

```suggestion
namespace Microsoft.CodeAnalysis.UnitTests.WorkspaceServices.Fakes
{
    internal class StubServiceBroker : IServiceBroker
```